models/signed-resource.js

```diff
@@ -1,75 +1,139 @@
 // @ts-ignore
-import {prefixMap} from "../support/prefixes";
-import {query, sparqlEscapeUri} from "mu";
+import { prefixMap } from "../support/prefixes";
+import { query, sparqlEscapeUri } from "mu";
 
 export default class SignedResource {
   static async findURI(uri) {
     const queryString = `
     ${prefixMap.get("mu").toSparqlString()}
     ${prefixMap.get("sign").toSparqlString()}
     ${prefixMap.get("dct").toSparqlString()}
-    SELECT ?uri ?uuid ?content ?signatory ?signatoryRole ?created ?hashValue ?hashAlgorithm
+    ${prefixMap.get("publicationStatus").toSparqlString()}
+    ${prefixMap.get("ext").toSparqlString()}
+    SELECT DISTINCT *
     WHERE {
       BIND(${sparqlEscapeUri(uri)} as ?uri)
         ?uri a sign:SignedResource;
-        mu:uuid ?uuid;
-        sign:text ?content;
-        sign:signatory ?signatory;
-        sign:signatoryRoles ?signatoryRole;
-        dct:created ?created;
-        sign:status publicationStatus:unpublished;
-        sign:hashAlgorithm ?hashAlgorithm;
-        sign:hashValue ?hashValue.
-        dct:subject ?subject.
-    }
+             mu:uuid ?uuid;
+             sign:text ?html;
+             sign:hashValue ?hashValue;
+             sign:hashAlgorithm ?hashAlgorithm;
+             dct:created ?created;
+             sign:signatoryRoles ?signatoryRole.
+        
+        ?uri sign:status ?blockchainStatus;
+             sign:signatory ?signatory.
+        
+        ?blockchainStatus mu:uuid ?blockchainStatusUuid.
```

Comment:
I'd blockchainStatus and uuid legacy and no longer required

Reply:
in the interest of limiting scope, OK to leave it for now and open a separate pr to clean up obsolete stuff?